### DISCUSSION TITLE
[Feature request] Hidden advanced options

### DISCUSSION THREAD

**Discussion by egvimo** [PaulWoitaschek/Voice#1321] [Ideas]
Since the app is developed according to the minimalistic principle, there are not many settings. However, there are some users who would still have some extra settings. For example, I would very much like to have an option for this: #486.

Therefore I suggest to create another hidden view that contains some advanced options. These could be activated in the same way as the android developer options. So tap 7 times on a certain element (e.g. the Support headline in the settings if you tap on the heart).

This way the minimalistic principle is kept and special wishes still can be fulfilled.

**Comment by MalakymR** [comment]
No need to have lack of options in a minimal interface, what they should really mean is the front end day to day use, not the settings panel(s). Why people think this is beyond me - they could just remove the background on the lock screen if they really want minimalism... but its there.

A better way of doing it, the Kodi way, is to have an option in the settings to say what mode you want, Minimal, Advanced, Expert. Each option should be tagged with what level it is and decide to show/hide it based on this.
